tests/unit/s2n_client_hello_retry_test.c

```diff
@@ -490,6 +492,12 @@ int main(int argc, char **argv)
         server_conn->x509_validator.skip_cert_validation = 1;
         client_conn->x509_validator.skip_cert_validation = 1;
 
+        /* include cookie data as part of HRR */
+        for(int i = 1; i < 500; ++i) {
+            EXPECT_SUCCESS(s2n_stuffer_write_uint8(&server_conn->cookie_stuffer, 0));
```

Comment:
Not sure if this belongs in this test case but maybe it's useful to add a test case to verify wiped `s2n_stuffers` in `s2n_connection` buffer's are set to `S2N_WIPE_PATTERN` after it is wiped. I didn't see an equivalent in `s2n_connection_test.c` but may be I missed it

Reply:
Also I think that could be a separate PR